### PR TITLE
Persian i18n added and translated

### PR DESCRIPTION
**Pull request by AdnanBabakan**

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -197,6 +201,13 @@ export default {
         file: 'pt-BR.js',
         name: 'Português',
         domain: 'https://pt.nuxtjs.org'
+      },
```
Comment (debs-obrien) [inline nuxt.config.js:204]: this lang cannot be put live yet until the subdomain has been created so for now you can leave it commented out.

File: nuxt.config.js [hunk 2]
```diff
@@ -2,6 +2,10 @@ export default {
   target: 'static',
   ssr: true,
   head: {
+    htmlAttrs: {
+      lang: 'fa',
```
Comment (debs-obrien) [inline nuxt.config.js:6]: this will be added to every single language if I am not mistaken so we need to add rtl only if the lang is fa.

**Review by debs-obrien — COMMENTED** [review]
Great start and nice adding the rtl. this is the first rtl lang we have for this site. Just a few comments I left regarding the nuxt config. To make it easier for others to collaborate I would just copy over the file you have translated and that means others in the issue can see which files/chapters are missing rather than having to open each md file to see if it needs translating. But just my opinion. It is what the others have been doing that are working on other languages and seems to be working quite well.

File: nuxt.config.js (hunk 1 above)
Comment (AdnanBabakan) [reply nuxt.config.js:204]: Sure

File: nuxt.config.js (hunk 2 above)
Comment (AdnanBabakan) [reply nuxt.config.js:6]: So how do you suggest to do it? What's the proper way to do so? Is there a way to get the current language and act according to that?

Comment (debs-obrien) [reply nuxt.config.js:6]: good question. we will have to try something using if i18n.locale === fa add rtl. Never tried it before but give it a go and if you get stuck just comment it out and we will look into it next week.

Comment (AdnanBabakan) [reply nuxt.config.js:6]: I tried this and it is working perfectly:

```javascript
    htmlAttrs: {
      lang: process.env.NUXT_LOCALE,
      dir: ['fa', 'ar', 'he'].includes(process.env.NUXT_LOCALE) ? 'rtl' : 'ltr'
    }
```

And for the production we can for sure generate the website using the local wanted.

The only three languages I know which are RTL are Persian, Arabic and Hebrew so I added them to the array.

Comment (debs-obrien) [reply nuxt.config.js:6]: fantastic. great job 💪

**Comment by debs-obrien** [comment]
so @AdnanBabakan  what do you think of removing the non translated files to make it easier for others to just copy one over and work on it rather than having them all there

**Comment by AdnanBabakan** [comment]
> so @AdnanBabakan what do you think of removing the non translated files to make it easier for others to just copy one over and work on it rather than having them all there

Of course. I already did it after I read your comment, the new commit will have the translated files only.

**Comment by debs-obrien** [comment]
Fantastic some great work done there. Well done. Just make sure the nuxt config has the lang commented out like the spanish language. By adding it here it adds it automatically to the footer select so we can't show it till domain is live. 
Other than that everything else looks great

**Comment by AdnanBabakan** [comment]
Sure

**Pull request merged**